packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -284,6 +285,16 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
       suggestedDisplayValue: itemValue,
       suggestionsVisible: this.input.value !== '' && this.input.inputElement === document.activeElement
     });
+
+    /**
+     * If user exits the input box before suggestions are returned,
+     * select the first result upon promise resolution, if a suggestion
+     * is available.
+     */
+    if (this.suggestionStore.hasSelectedSuggestion() &&
```

Comment:
Would you be open to making this is a behavior that you can opt in to? This behavior is exactly how our people picker has worked in the past and how customers are expecting it to work now and why we are receiving escalations and negative feedback.